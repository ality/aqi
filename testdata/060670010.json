{"siteName":"Downtown Sacramento - T Street","stationID":"060670010","fullAQSCode":"060670010","intlCode":"840060670010","coordinates":[-121.49311,38.56844],"monitors":[{"parameterName":"O3","parameterDisplayName":"O3 - 44201","concUnit":"PPB","aqi":[32.0,31.0,30.0,27.0,24.0,23.0,22.0,22.0,21.0,23.0,20.0,19.0,19.0,17.0,18.0,19.0,18.0,21.0,24.0,28.0,34.0,34.0,32.0,32.0],"conc":[36.0,31.0,29.0,28.0,25.0,24.0,24.0,24.0,25.0,26.0,26.0,25.0,-999.0,17.0,16.0,16.0,17.0,20.0,25.0,32.0,41.0,43.0,43.0,-999.0]},{"parameterName":"PM2.5 - Principal","parameterDisplayName":"PM2.5 - Principal - PM25","concUnit":"UG/M3","aqi":[35.0,36.0,37.0,35.0,26.0,23.0,22.0,30.0,31.0,26.0,23.0,24.0,23.0,34.0,34.0,27.0,20.0,22.0,32.0,62.0,56.0,55.0,50.0,50.0],"conc":[7.0,9.0,9.0,8.0,4.0,5.0,5.0,9.0,8.0,5.0,5.0,6.0,5.0,11.0,8.0,5.0,3.0,6.0,10.0,27.0,12.0,13.0,10.0,-999.0]},{"parameterName":"PM10 - Principal","parameterDisplayName":"PM10 - Principal - PM10 ","concUnit":"UG/M3","aqi":[40.0,40.0,43.0,31.0,23.0,24.0,20.0,16.0,14.0,11.0,11.0,10.0,11.0,11.0,13.0,19.0,19.0,20.0,21.0,32.0,40.0,37.0,32.0,32.0],"conc":[42.0,42.0,49.0,22.0,16.0,28.0,18.0,12.0,13.0,9.0,12.0,11.0,13.0,11.0,17.0,27.0,19.0,25.0,24.0,47.0,52.0,37.0,31.0,-999.0]}],"utcOffset":-7.0,"startTimeUTC":"2021-08-13 00:00:00","endTimeUTC":"2021-08-13 23:00:00","utcDateTimes":["2021-08-13 00:00:00","2021-08-13 01:00:00","2021-08-13 02:00:00","2021-08-13 03:00:00","2021-08-13 04:00:00","2021-08-13 05:00:00","2021-08-13 06:00:00","2021-08-13 07:00:00","2021-08-13 08:00:00","2021-08-13 09:00:00","2021-08-13 10:00:00","2021-08-13 11:00:00","2021-08-13 12:00:00","2021-08-13 13:00:00","2021-08-13 14:00:00","2021-08-13 15:00:00","2021-08-13 16:00:00","2021-08-13 17:00:00","2021-08-13 18:00:00","2021-08-13 19:00:00","2021-08-13 20:00:00","2021-08-13 21:00:00","2021-08-13 22:00:00","2021-08-13 23:00:00"],"fileWrittenDateTime":"20210814T001238Z"}