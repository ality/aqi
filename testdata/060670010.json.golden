measurement start time   	pm25 aqi         	pm10 aqi         	ozone aqi
2021-08-13T05:00:00-07:00	G 23 (5 µg/m³)   	G 11 (13 µg/m³)  	G 19 (- ppb)
2021-08-13T06:00:00-07:00	G 34 (11 µg/m³)  	G 11 (11 µg/m³)  	G 17 (17 ppb)
2021-08-13T07:00:00-07:00	G 34 (8 µg/m³)   	G 13 (17 µg/m³)  	G 18 (16 ppb)
2021-08-13T08:00:00-07:00	G 27 (5 µg/m³)   	G 19 (27 µg/m³)  	G 19 (16 ppb)
2021-08-13T09:00:00-07:00	G 20 (3 µg/m³)   	G 19 (19 µg/m³)  	G 18 (17 ppb)
2021-08-13T10:00:00-07:00	G 22 (6 µg/m³)   	G 20 (25 µg/m³)  	G 21 (20 ppb)
2021-08-13T11:00:00-07:00	G 32 (10 µg/m³)  	G 21 (24 µg/m³)  	G 24 (25 ppb)
2021-08-13T12:00:00-07:00	M 62 (27 µg/m³)  	G 32 (47 µg/m³)  	G 28 (32 ppb)
2021-08-13T13:00:00-07:00	M 56 (12 µg/m³)  	G 40 (52 µg/m³)  	G 34 (41 ppb)
2021-08-13T14:00:00-07:00	M 55 (13 µg/m³)  	G 37 (37 µg/m³)  	G 34 (43 ppb)
2021-08-13T15:00:00-07:00	G 50 (10 µg/m³)  	G 32 (31 µg/m³)  	G 32 (43 ppb)
2021-08-13T16:00:00-07:00	G 50 (- µg/m³)   	G 32 (- µg/m³)   	G 32 (- ppb)
